### PR TITLE
Xot 1224 market landing

### PR DESCRIPTION
**Pull request by jeanettemariemurphy**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if adding CSS) CSS have been compiled.
 - [x] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [x] (if changing the UI) Add a printscreen to the PR

screencapture-exred-trade-great-8007-markets-2020-01-09-12_19_51

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -8,12 +8,19 @@
...
+from directory_components.context_processors import urls_processor
+
 
 @pytest.fixture(autouse=True)
 def clear_django_cache():
     cache.clear()
 
 
+@pytest.fixture(autouse=True)
+def context():
```
Comment (richtier) [inline conftest.py]: as far as I can tell this fixture is used in only a few tests. I would instead move this to the tests:

```
context = urls_processor(None)
```

simpler :)

File: content/templates/content/markets_landing_page.html [hunk 2]
```diff
@@ -2,57 +2,165 @@
...
+                      {% endfor %}
+                    </ul>
+                  </div>
+              </li>
 
-    {% if number_of_results > 0 and tag_name %}
-    <h2 class="font-medium margin-bottom-45">Best markets for <span class="bold">{{ tag_name }}</span> <span class="number-display highlight-circle background-active-blue">{{ number_of_results }}</span></h2>
+            </ul>
+            <div class="filters-sticky-footer" id="filters-sticky-footer">
+              <a href="#" class="cancel link">Cancel</a>
```
Comment (richtier) [inline content/templates/content/markets_landing_page.html]: do these buttons with `href="#"` do anything?

Comment (jeanettemariemurphy) [reply content/templates/content/markets_landing_page.html]: Yes, they are for the mobile view only though, they are bound to the event handlers clearMobileFilters and submitForm in the javascript.  The mobile view has the filters as a fullpage overlay, so cancel button closes that and the other submits the form.  The mobile form has a different behaviour to the desktop form in this way.

File: conftest.py (hunk 1 above)
Comment (jeanettemariemurphy) [reply conftest.py]: This was added by @eadpearce originally as a fix to a components change - I merged this into my branch - would you be happy for this change be done retrospectively if it is needed?

Comment (jeanettemariemurphy) [reply conftest.py]: I have update this now - please can you recheck? thanks

**Pull request merged**